.gitignore

```diff
@@ -1,2 +1,3 @@
 /node_modules
 npm-debug.log*
+.idea/*
```

Comment:
`.idea` should be ignored. Remove `+`.

Reply:
Although out of curiosity, what is `idea` ?

Reply:
Idea contains some web-storm configurations.